src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -219,6 +222,10 @@ private static void analyzeRepo(
 
         CommitContributionSummary commitSummary = CommitsReporter.generateCommitSummary(config);
         AuthorshipSummary authorshipSummary = AuthorshipReporter.generateAuthorshipSummary(config);
+
+        // updates path of the local repo so that can be viewed in browser
+        updateRepoPath(config);
```

Comment:
I think currently, we either believe that the matcher matches or otherwise its a local repo. So, maybe we can create a new issue and do a  different PR to handle the case which you have just suggested, and let this PR just handle links of the local repositories. 

> Why not append `file:///` in RepoLocation?

For this, I thought it is better to update during the report generation because it will be creating the link at that time, but this makes the logic complex. Will move forward with your suggestion.